### PR TITLE
[FrameworkBundle] fix assets and templating tests

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #17506 |
| License | MIT |
| Doc PR |  |

Now that the `assets` section is automatically enabled when `templating`
is enabled (see #17506), the `templating.helper.assets` will also be
present as soon as the `templating` section is activated.

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -547,11 +547,7 @@ private function registerTemplatingConfiguration(array $config, $ide, ContainerB
                 'Symfony\\Bundle\\FrameworkBundle\\Templating\\Loader\\FilesystemLoader',
             ));
 
-            if ($container->hasDefinition('assets.packages')) {
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: isn't it still possible to disable the asset system if we don't need it ?

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: That would mean to revert #17506.

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php [hunk 2]
```diff
@@ -438,7 +438,15 @@ public function testAssetHelperWhenAssetsAreEnabled()
 
     public function testAssetHelperWhenTemplatesAreEnabledAndAssetsAreDisabled()
     {
-        $container = $this->createContainerFromFile('assets_disabled');
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php]: shouldn't we remove also `src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/assets_disabled.*`?

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php]: Yes, good catch. Thanks!

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**